cases/forms/case.py

```diff
@@ -115,6 +119,14 @@ class Meta:
         label="Internal notes",
         help_text="You can upload documents by adding a link or links to shared documents",
     )
+    initial_time = localtime(now())
```

Comment:
This value will be set once at code import time. If the code is then held in memory, it'll always be the same time, regardless of how many times the form is used. I'm not sure if this applies in our actual usage, but I think it might - even if it isn't, we should evaluate this at run-time. You could have a function that you then put as the initial callable; and same for the action_time.